### ISSUE TITLE
VG autoindex giraffe-workflow and genotype

### ISSUE THREAD

**Issue by wjwei-handsome** [vgteam/vg#3880]
Hi VGTeam!

I' trying to genotype some SVs using such pipeline: `autoindex` , `giraffe`, `pack`, `call`. 

I noticed that `GBZ` format graph could be used in all steps. So I tried like this:

1. `vg autoindex --prefix my-test --workflow giraffe --ref-fasta ref.fa --vcf ins.vcf.gz --threads 8`
>As excepted, the `gbz`,`min`,`dist` index files were created.
2. `vg giraffe -Z my-test.giraffe.gbz -m my-test.min -d my-test.dist -f rd1.fq.gz -f rd2.fq.gz -t 8 -N sample > sample.gam`
3. `vg pack -x my-test.giraffe.gbz  -t 8 -g sample.gam -o sample.pack -Q 5`
>I used `gbz` graph for pack

4.`vg snarls my-test.giraffe.gbz --threads 4 > my-test.snarls`
5.`vg call my-test.giraffe.gbz -z -k sample.pack -s sample -t 8 -r my-test.snarls  -v ins.vcf.gz > sample.geno.vcf`

But when i trying to genotype SVs in vcf that was constructed in graph, the caller just report waring like this:

`[VCFTraversalFinder] Warning: No alt path *** found in graph for variant.  It will be ignored:`

These warnings are critical fro genotyping.

I am using vg version v1.46.0 "Altamura".

Thanks for your help.

PS: I also tried using classical pipeline looks like:

1. `vg construct --reference genome.fa --vcf ins.vcf.gz --alt-paths --flat-alts --handle-sv --progress --threads 4 > all.vg`
2. `vg gbwt --xg-name all.vg --progress --num-jobs 10 --vcf-input all.ins.vcf.gz --vcf-variants --graph-name all.gbz --gbz-format`
3. `vg index --threads 16 --progress --dist-name all.dist all.gbz`

But in the step3, VG crashed as reported:
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  error: trying to access a snarl tree node of the wrong type
ERROR: Signal 6 occurred. VG has crashed. Visit https://github.com/vgteam/vg/issues/new/choose to report a bug.
```

In summary, I want to know how to build a graph for structural variation, and genotyping using short-reads by `giraffe`-`pack`-`call`.

Thanks!!

**Comment by smriti-135** [comment]
Hello! I am facing same issues. Can you tell me the workflow you followed to solve it? I am trying to do pangenome construction using short-read sequences. I have tried many methods (autoindex vs manual, gbz vs vg vs xg vs pg) and still not getting the SVs.

**Comment by wjwei-handsome** [comment]
Please refer this issue:
https://github.com/vgteam/vg/issues/3884
